crypto/objects/search.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 1995-2019 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/objects.h>
+
+const void *OBJ_bsearch_(const void *key, const void *base, int num, int size,
```

Comment:
I don't recall seeing trailing underscore; isn't there another in-use convention you could use?

Reply:
Moving this code around turned out to be unnecessary so I've removed it from the scope of this PR anyway.